packages/core/src/services/fileconfiguration.ts

```diff
@@ -37,6 +37,13 @@ export class FileConfigurationService<
    * @override
    */
   protected async loadConfiguration(): Promise<{ [key: string]: any }> {
+    /**
+     * Auto-generate file if missing
+     */
+    if (!existsSync(this.parameters.source) && this.parameters.default) {
+      writeFileSync(this.parameters.source, JSON.stringify(this.parameters.default, null, 2));
```

Comment:
You can use `JSONUtils.save(this.parameteres.default, this.parameters.source)`

Reply:
you mean `saveFile`